website/docs/getting-started/index.mdx

```diff
@@ -0,0 +1,560 @@
+up-to-date as follows:
+
+
+<MDXTabs namespace={`getting-started-front-end-codetabs`}>
+
+
+<MDXTab label="URQL React">
+
+```tsx
+import { useQuery } from "urql";
+import { postsQueryDocument } from './graphql/generated';
+
+const Posts = () => {
+  const [result] = useQuery({ query: postsQueryDocument});
+
+  // `result` is fully typed!
+  // ...
+}
+```
+
+</MDXTab>
+
+
+<MDXTab label="React Query">
+
+```tsx
+import { usePosts } from "../graphql/generated";
+
+
+const Posts = () => {
+  const { data } = usePosts()
+
+  // `data` is typed!
+
+  // ...
+}
+```
+
+</MDXTab>
+
+<MDXTab label="Vue Apollo">
+
+```vue
+<template>
+  <div class="apollo">
+    <!-- UI... -->
+  </div>
+</template>
+
+<script>
+import { usePostQuery } from '../generated/graphql';
+
+export default {
+  apollo: usePostQuery({
+    // `variables` is properly typed!
+    variables: {
+      id: 1
+    }
+  }),
+  data () {
+    return {
+      post: undefined,
+    }
+  },
+};
+</script>
+```
+
+</MDXTab>
+
+<MDXTab label="Angular Apollo">
+
+```tsx
+import { PostsGQL, PostsQuery } from './graphql'
+//BE SURE TO USE Observable from `rxjs` and not from `@apollo/client/core` when using map
+import { Observable } from 'rxjs'
+import { map } from 'rxjs/operators'
+
+@Component({ /* ... */ })
+export class PostsComponent {
+  posts: Observable<PostsQuery['posts']>
+
+  constructor(postsGQL: PostsGQL) {
+    this.posts = postsGQL.watch().valueChanges.pipe(map(result => result.data.posts))
+  }
+}
+```
+
+</MDXTab>
+
+<MDXTab label="Svelte Apollo">
+
+```svelte
+<script lang="ts">
+  import { Posts } from "../graphql/generated";
+
+  // `posts` is fully typed, also are `Posts()` options!
+  const posts = Posts();
+</script>
+
+<ul>
+    { /* UI */ }
+</ul>
+```
+
+</MDXTab>
+
+</MDXTabs>
+
+Now, with a simple configuration and an npm/yarn script, the front-end developers benefit from:
+
+
+  - **up-to-date typings**
+
+
+  - **autocompletion on all queries, mutations and, subscription variables and results**
+
+
+  - **less boilerplate** (thanks to full code generation such as React hooks generation)
+
+
+
+<br />
+
+:::tip How does GraphQL Code Generator work?
+More details on the inner working of GraphQL Code Generator are available on [this page](/docs/getting-started/how-does-it-work).
+:::
+
+
+<p>&nbsp;</p>
+
+----
+
+<p>&nbsp;</p>
+
+### To the back-end
+
+Most GraphQL API resolvers remain untyped or wrongly typed which, leads to multiple issues:
+
+- resolvers not compliant with the Schema definition
+
+- unhandled scenarios
+
+- typos in the resolvers' function type signature
+
+<br/>
+
+For this reason, GraphQL Code Generator provides multiple plugins that help automate the generation of resolvers' typings.
+
+Here are an example of a GraphQL API leveraging GraphQL Code Generator resolvers typings (based on the `schema.graphql` above):
```

Comment:
Similar to the front-end section it could be nice to also show an example of untyped resolvers.

Reply:
I feel like the code differences between without and with codegen on the back-end are less visual, therefore, adding code examples like on the front-end is not bringing much.

wdyt?